test/sqlancer/reducer/VirtualDB/VirtualDBOptions.java

```diff
@@ -0,0 +1,27 @@
+package sqlancer.reducer.VirtualDB;
+
+import sqlancer.DBMSSpecificOptions;
+import sqlancer.OracleFactory;
+import sqlancer.common.oracle.TestOracle;
+import sqlancer.reducer.VirtualDB.VirtualDBOptions.VirtualDBFactory;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VirtualDBOptions implements DBMSSpecificOptions<VirtualDBFactory> {
+
+    List<VirtualDBFactory> factories = new ArrayList<>();
+
+    @Override
+    public List<VirtualDBFactory> getTestOracleFactory() {
+        return factories;
+    }
+
+    public enum VirtualDBFactory implements OracleFactory<VirtualDBGlobalState> {
```

Comment:
Do we actually need this enum? Could we just return an empty list above?

Reply:
OK, I would change it to a dummy class.